### ISSUE TITLE
Testing assertion errors, probably due to quotation marks

### ISSUE THREAD

**Issue by TheChymera** [click-contrib/click-didyoumean#11]
I am getting the following errors when running the test suite: https://ppb.chymera.eu/a76304

I assume this is because of quotation mark mismatch. Is this something that should be fixed here?

**Comment by timofurrer** [comment]
This particular line "No such command ..." is printed by click itself.
What version of click on which platform are you using?

**Comment by TheChymera** [comment]
8.0.1 on Gentoo Linux

**Comment by timofurrer** [comment]
I've just noticed that you've changed the test code which leads to this issue.
click uses `__repr__` to print that missing command (https://github.com/pallets/click/blob/e8e4b32f5034b8e6df9f4d5ba7ed952da9394a76/src/click/core.py#L1715) which uses single quotes.

You've changed it to double quotes ;)

see 

https://github.com/click-contrib/click-didyoumean/blob/master/tests/test_core.py#L40

and yours:

![image](https://user-images.githubusercontent.com/1008252/134899622-a39ab006-9391-4632-b452-41a214a073f3.png)

**Comment by TheChymera** [comment]
@timofurrer I didn't. In fact, it seems the issue is that you're looking at the live version, not at the most recent release: https://github.com/click-contrib/click-didyoumean/blob/80913ac6907d531346ce98e5902bf3a5707d5bda/tests/test_core.py — which has double quotes.

It also appears the “most recent release” is rather ancient. Do you have some newer versioning scheme?

**Comment by timofurrer** [comment]
I see :)

> It also appears the “most recent release” is rather ancient. Do you have some newer versioning scheme?

Yes, the project here can use some love and the latest release is quite out dated. 

I suggest that you use it from the `master` branch for now. Let's see if I can find a few hours this week to bring it up to date :)

**Comment by TheChymera** [comment]
@timofurrer Merci vielmohls ;)

**Comment by timofurrer** [comment]
A first release 0.1.0 is on PyPI. More updates follow :)

Cheers! 🇨🇭

**Comment by TheChymera** [comment]
@timofurrer I tried both the `0.1.0` and `0.2.0` versions you released, and I get the following for both:

```
 * python3_9: running distutils-r1_run_phase python_test
python3.9 -m pytest -vv -ra -l -Wdefault
=========================================================== test session starts ============================================================
platform linux -- Python 3.9.7, pytest-6.2.5, py-1.10.0, pluggy-1.0.0 -- /usr/bin/python3.9
cachedir: .pytest_cache
rootdir: /var/tmp/portage/dev-python/click-didyoumean-0.2.0/work/click-didyoumean-0.2.0
plugins: pyfakefs-4.5.1, timeout-1.4.2, mock-3.6.1
collecting ... collected 0 items

========================================================== no tests ran in 0.01s ===========================================================
```

**Comment by TheChymera** [comment]
@timofurrer the tests pass from the live cloned repo in any case (they always have, this was just regarding the releases). But it seems the tests aren't included in the PyPI tarball:

```
chymera@decohost ~/click-didyoumean-0.2.0 $ tree
.
├── LICENSE
├── PKG-INFO
├── pyproject.toml
├── setup.py
└── src
    └── click_didyoumean
        └── __init__.py

2 directories, 5 files
```

Could you include them, or also release the respective versions via GitHub? :)

**Comment by TheChymera** [comment]
@timofurrer tests are now included and pass as of the `0.3.0` GitHub tarball.
Thank you :)
